Mapping/grid_map_lib/grid_map_lib.py

```diff
@@ -120,8 +120,8 @@ def set_value_from_polygon(self, pol_x, pol_y, val, inside=True):
 
         # making ring polygon
         if (pol_x[0] != pol_x[-1]) or (pol_y[0] != pol_y[-1]):
-            pol_x.append(pol_x[0])
-            pol_y.append(pol_y[0])
+            np.append(pol_x, pol_x[0])
```

Comment:
Thank you for your PR. is it possible to add a test for this change in this file? https://github.com/AtsushiSakai/PythonRobotics/blob/master/tests/test_grid_map_lib.py

Reply:
After my survey, the error occurs in `grid_based_sweep_coverage_path_planner.py` because the function `convert_grid_coordinate` returns `numpy.ndarray`. I think it more suitable to modify `grid_based_sweep_coverage_path_planner.py`:

from
```python
def convert_grid_coordinate(ox, oy, sweep_vec, sweep_start_position):
    tx = [ix - sweep_start_position[0] for ix in ox]
    ty = [iy - sweep_start_position[1] for iy in oy]
    th = math.atan2(sweep_vec[1], sweep_vec[0])
    rot = Rot.from_euler('z', th).as_matrix()[0:2, 0:2]
    converted_xy = np.stack([tx, ty]).T @ rot

    return converted_xy[:, 0], converted_xy[:, 1]
```
to
```python
def convert_grid_coordinate(ox, oy, sweep_vec, sweep_start_position):
    tx = [ix - sweep_start_position[0] for ix in ox]
    ty = [iy - sweep_start_position[1] for iy in oy]
    th = math.atan2(sweep_vec[1], sweep_vec[0])
    rot = Rot.from_euler('z', th).as_matrix()[0:2, 0:2]
    converted_xy = np.stack([tx, ty]).T @ rot

    return converted_xy[:, 0].tolist(), converted_xy[:, 1].tolist()
```
What do you think about it?

Reply:
I prefer to use np.array instead of build-in list in codes. So I love this PR change.

Reply:
Alright. I think test should be add to [https://github.com/AtsushiSakai/PythonRobotics/blob/master/tests/test_grid_based_sweep_coverage_path_planner.py](https://github.com/AtsushiSakai/PythonRobotics/blob/master/tests/test_grid_based_sweep_coverage_path_planner.py) because in [https://github.com/AtsushiSakai/PythonRobotics/blob/master/tests/test_grid_map_lib.py](https://github.com/AtsushiSakai/PythonRobotics/blob/master/tests/test_grid_map_lib.py), `ox[0] != ox[-1]` and `oy[0]!=oy[-1]` make `np.append` effective.
```python
def test_polygon_set():
    ox = [0.0, 20.0, 50.0, 100.0, 130.0, 40.0]
    oy = [0.0, -20.0, 0.0, 30.0, 60.0, 80.0]

    grid_map = GridMap(600, 290, 0.7, 60.0, 30.5)

    grid_map.set_value_from_polygon(ox, oy, 1.0, inside=False)
```

Reply:
The test looks good for me!